src/rules.jl

```diff
@@ -518,6 +530,28 @@ function lrp!(Rᵏ, rule::GeneralizedGammaRule, layer, modified_layers, aᵏ, R
     @. Rᵏ = aᵏ⁺ * (cˡ⁺ + cʳ⁻) + aᵏ⁻ * (cˡ⁻ + cʳ⁺)
 end
 
+"""
+#
+"""
+function lrp!(Rᵏ, rule, layer::LayerNorm, modified_layer, aᵏ, Rᵏ⁺¹)
+    # forward pass: split in normalization and scale
+    # aᵏ ->(normalize) aᵏₙ ->(scale) aᵏ⁺¹
+    ## normalize
+    n_dims = 1:length(layer.size)
+    eps = convert(float(eltype(aᵏ)), layer.ϵ)
+    μ = mean(aᵏ; dims=n_dims)
+    σ = std(aᵏ, dims=n_dims, mean=μ, corrected=false)
+    z = aᵏ - μ
+    aᵏₙ = @. z / (σ + eps)
+    # lrp pass
+    ## Rᵏ⁺¹ ->(scale) Rᵏ ->(normalize) Rᵏ
+    ## scale: call LRP on affine layer with "subrule" rule.affine_rule
+    lrp!(Rᵏ, rule, layer.diag, modify_layer(rule, layer.diag), aᵏₙ, Rᵏ⁺¹)
```

Comment:
@adrhill: so it's okay for now that we can't pass `keep_bias` in this call to `modify_layer`?

Reply:
Yes!